aea/crypto/wallet.py

```diff
@@ -91,8 +91,6 @@ class Wallet:
 
     """
 
-    # TODO do some check after loading the keys
```

Comment:
if we ever want to do extra checks we can add a new issue; atm this is not a priority